api/v1/dspipeline_types.go

```diff
@@ -69,9 +69,13 @@ type APIServer struct {
 	// Include sample pipelines with the deployment of this DSP API Server. Default: true
 	// +kubebuilder:default:=false
 	// +kubebuilder:validation:Optional
-	EnableSamplePipeline bool   `json:"enableSamplePipeline"`
-	ArgoLauncherImage    string `json:"argoLauncherImage,omitempty"`
-	ArgoDriverImage      string `json:"argoDriverImage,omitempty"`
+	EnableSamplePipeline bool `json:"enableSamplePipeline"`
+	// Include instructlab multi-phase training and LLM generation pipeline with the deployment of this DSP API Server. Default: false
```

Comment:
Since this is user facing documentation in the CRD, we should fix the casing. Sorry for missing this before.
```suggestion
	// Enable the Instructlab Multi-Phase Training pipeline with the deployment of this DSP API server. Default is false.
```